test/notifications/display_test.dart

```diff
@@ -79,6 +83,21 @@ void main() {
   TestZulipBinding.ensureInitialized();
   final zulipLocalizations = GlobalLocalizations.zulipLocalizations;
 
+  makeFakeHttpClient({http.Response? response, Exception? exception}) => http_testing.MockClient((request) async {
```

Comment:
nit: explicit return type for function declaration, so it doesn't look like a function call

Reply:
Looks like there is indeed a lint rule for this, `always_declare_return_types`:
https://dart.dev/tools/linter-rules/always_declare_return_types

I'll see about turning that on, since it keeps coming up.

Reply:
separate nit: use block body:
https://github.com/flutter/flutter/blob/main/docs/contributing/Style-guide-for-Flutter-repo.md#consider-using--for-short-functions-and-methods

Reply:
> I'll see about turning that on, since it keeps coming up.

→ #851